tests/dummy/app/templates/signup-example.hbs

```diff
@@ -1,20 +1,20 @@
 <h1>Signup example</h1>
 
-<form {{action "signup" on="submit"}}>
+<form class="signup-example-form" {{action "signup" on="submit"}}>
```

Comment:
Ah, I see. I didn't consider it. I wanted to create an example with async behaviour (loading substate, disable a button while a request is ongoing...) that is hard to test currently to prove that this approach would allow it. 

I think that we can keep them separated.